content/cpp/concepts/math-functions/terms/trunc/trunc.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'trunc()' 
+Description: 'Returns the integral part of the argument.' 
+Subjects:
+  - 'Computer Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`trunc()`** function returns the integral part of the argument. It rounds the argument towards zero and returns the nearest integer that is not greater in magnitude than the argument.
```

Comment:
The second sentence should be rewritten to take into account negative numbers. If a negative number is rounded toward zero, it invalidates the second clause because the result would be greater than the argument.